packages/contracts-bedrock/deploy/005-AddressManager.ts

```diff
@@ -0,0 +1,22 @@
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+import 'hardhat-deploy'
+import '@eth-optimism/hardhat-deploy-config'
+
+// TODO(tynes): This needs to be deployed for fresh networks
+// but not for upgrading existing networks
```

Comment:
What does this TODO mean in practice?

Reply:
This means that we need to configure this to run in "upgrade mode" and "devnet mode", so that it attaches to the existing address manager instead of deploying it in the code